.travis.yml

```diff
@@ -1,3 +1,6 @@
+arch:
+   - amd64
+   - ppc64le
```

Comment:
Thanks, but travis is obsolete. Please update GitHub workflow instead (`.github/workflows/ci.yaml`).

Reply:
In fact, `.travis.yml` doesn't even exist since 0cd847579f78c19132ac808c409edf7e692a6ded

Reply:
Hi @AlexanderAmelkin, Github actions don't support ppc64le currently and we will revisit this once that is fully supported.

Thanks !!